Libplanet.Net.Tests/Consensus/ContextTest.cs

```diff
@@ -867,6 +869,603 @@ public async void PreCommitBlockToEndCommit()
             Assert.True(blockChain.ContainsBlock(block.Hash));
         }
 
+        [Fact(Timeout = Timeout)]
+        public async void RoundSkipTimeoutProposeToPreCommitNilToPropose()
+        {
+            var (validators, privateKeys) = GetRandomValidators();
+
+            using ITransport transport =
+                TestUtils.CreateNetMQTransport(privateKeys[3], port: Port + 12);
+            var codec = new Codec();
+            BlockChain<DumbAction> blockChain =
+                TestUtils.CreateDummyBlockChain((MemoryStoreFixture)_fx);
+
+            var consensusContext =
+                TestUtils.CreateStandaloneConsensusContext(
+                    blockChain,
+                    transport,
+                    newHeightDelay,
+                    3,
+                    blockChain.Tip.Index + 1,
+                    port: Port + 12,
+                    privateKey: privateKeys[3],
+                    validators: validators);
+
+            _ = transport.StartAsync();
+            await transport.WaitForRunningAsync();
+
+            var context = new Context<DumbAction>(
+                consensusContext,
+                blockChain,
+                0,
+                blockChain.Tip.Index + 1,
+                privateKeys[3],
+                validators);
+            var block = await blockChain.MineBlock(privateKeys[3], append: false);
+            var timeoutOccurred = new AsyncManualResetEvent();
+            context.TimeoutOccurred += (sender, tuple) => timeoutOccurred.Set();
+
+            _ = transport.StartAsync();
+            await transport.WaitForRunningAsync();
```

Comment:
it might be called` ITransport.StartAsync()` twice in one instance, Is it intended?

Reply:
I'm sorry but, I don't understand when will `transport` called twice. I had to call `StartAsync()` without async and await `WaitForRunningAsync()` instead because awaiting `StartAsync()` would block the test. 😅

Reply:
Oh.. I see, my mistake.